testing/test-utils/src/lib/utils/minimal-config.mock.ts

```diff
@@ -50,7 +50,7 @@ export const MINIMAL_CONFIG_MOCK: CoreConfig = {
   plugins: [MINIMAL_PLUGIN_CONFIG_MOCK],
 };
 
-export const MINIMAL_HISTORY_CONFIG_MOCK: CoreConfig = {
```

Comment:
Why remove the type annotation? :raised_eyebrow:

Reply:
added back